### DISCUSSION TITLE
Toggle bar visibility on hotkey.

### DISCUSSION THREAD

**Discussion by lominoss-git** [qtile/qtile#3013] [Q&A] [answered]
I'm just looking for a way to hide and unhide the Qtile bar on hotkey.

**Comment by m-col** [comment, marked as answer]
There is a command for this, see: http://docs.qtile.org/en/latest/manual/ref/commands.html?highlight=show_hide#libqtile.core.manager.Qtile.cmd_hide_show_bar

Example usage:
```py
Key(mods, "b", lazy.hide_show_bar(position="top"))
```

**Reply by lominoss-git** [reply]
Thank you so much my dude
